coral/src/services/api.ts

```diff
@@ -25,12 +29,165 @@ type SomeObject =
   | Record<string, never>
   | Array<unknown>;
 
-type AbsolutePathname = `/${string}`;
-
 const CONTENT_TYPE_JSON = "application/json" as const;
 
 const API_BASE_URL = getHTTPBaseAPIUrl();
 
+const API_PATHS: { [key in keyof ApiOperations]: keyof ApiPaths } = {
```

Comment:
I wonder if we can find some way (in the future!!) to auto-generate that map from the openapi file 🤔  
(that sounds like a super fun problem to puzzle around with 🥹 but it's really not needed right now :D )

Reply:
I think such a map could probably exist in the Java code used to generate the OpenApi spec, and then we could use it directly. But I don't know what work it entails exactly ^^